core/src/main/java/io/questdb/griffin/engine/table/LatestByValuesFilteredRecordCursor.java

```diff
@@ -89,5 +93,10 @@ private void prepare() {
         for (int i = 0, n = symbolKeys.size(); i < n; i++) {
             map.put(symbolKeys.get(i), 0);
         }
+        if (deferredSymbolKeys != null) {
```

Comment:
also suspicious why map is never cleaned

Reply:
Yes, it's the same problem as in `LatestByValuesFilteredRecordCursor`. Fixed in dadeded966de06cedbbe5ccfbfe2d163569aa027